### PR TITLE
[Metis] clean includes

### PR DESCRIPTION
**Pull request by philbucher**
same as #3696 

plus adding the app-logo

### CODE REVIEW

File: applications/metis_application/custom_processes/metis_scalar_reorder_process.h [hunk 1]
```diff
@@ -11,18 +11,12 @@
...
 
-
 // System includes
-#include <string>
-#include <iostream>
-
 
 // External includes
-
+#include <parmetis.h>
```
Comment (RiccardoRossi) [inline applications/metis_application/custom_processes/metis_scalar_reorder_process.h:17]: i see that it is not you changing this...however the dependency should be on metis.h NOT on parmetis.h

**Review by RiccardoRossi — APPROVED** [review]
approving, nevertheless we should remove (in a difference PR) dependency on parmetis.h

File: applications/metis_application/custom_processes/metis_scalar_reorder_process.h (hunk 1 above)
Comment (philbucher) [reply applications/metis_application/custom_processes/metis_scalar_reorder_process.h:17]: I will try to do this in a separate PR

Comment (philbucher) [reply applications/metis_application/custom_processes/metis_scalar_reorder_process.h:17]: Hm this seems not to be trivial, apparently for metis-version < 5 parmetis is included
I don't know how/why this is done so I would prefer if someone else does to not break anything

**Pull request merged**